### PR TITLE
T-FilesToRDF: Manual Select of Graph Type

### LINKED ISSUE #350: t-filesToRdf: brittle format detection
It seems that t-filesToRdf detects the format of the input RDF file by looking at its suffix. This is a brittle way of format detection.

For example, when I use e-filesDownload to retrieve an RDF file from local file system or the web and I leave the "File name" empty, t-filesToRdf fails with `org.openrdf.rio.UnsupportedRDFormatException: No parser factory available for RDF format null`. When I download `https://raw.githubusercontent.com/opendatacz/public-contracts-ontology/master/public-contracts.ttl`, I expect that e-filesDownload will name its output file `public-contracts.ttl` and the `*.ttl` suffix will be recognized by t-filesToRdf, however that is not the case (it seems that this was introduced when #274 was fixed).

I think a pragmatic solution to this issue is to allow to configure the format of the input in t-filesToRdf.

Context: I compiled t-filesToRdf from the latest develop commit (33ceb3dbcf4bf0499e50783c5c0b41b6ecbc36ff).

### PR DESCRIPTION
**Pull request by mbl-eea-sk**
Now the Vaadin Dialog contains a combo box with choice of all known mimetypes that can be manually selected by user.
Resolves #350

### CODE REVIEW

File: t-filesToRdf/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 T-FilesToRdf
 ----------
 
+v2.2.0
```
Comment (skrchnavy) [inline t-filesToRdf/CHANGELOG.md]: Shall be sync with pom.xml, so SNAPSHOT version

**Comment by tomas-knap** [comment]
Currently the configuration for the type of graph is the last conf and at the same row with "Policy for output symbolic name selection", but it is not anyhow related with that policy. 

screen shot 2015-10-01 at 10 50 34

Please move the configuration for "Type of Graph" as the first field in the configuration.

**Comment by tomas-knap** [comment]
Also if I specify wrong RDF data format, the error message is rather confusing, it says (in this case I am loading ttl file, but I specifyed rdf+xml: 

screen shot 2015-10-01 at 11 06 26

Please improve the error message, so that it says e.g.:

**Comment by tomas-knap** [comment]
Instead of giving the user to possibility to choose from Mime Types, I suggest to use directly RDFFormat values in the combo box - http://rdf4j.org/sesame/2.8/apidocs/org/openrdf/rio/RDFFormat.html - , in particular the combo box should show : 
* RDF/XML (which is mapped to RDFFormat.RDFXML)
* TURTLE (which is mapped to RDFFormat.TURTLE)
* N3
* RDF/JSON (RDFFormat.RDFJSON)
* TRIG 
* TRIX
* NTRIPLES
* NQUADS
* JSON/LD

The motivation is that using MIME types is a bit confusing. For example, text/plain and also 
application/n-triples maps to n-triples, but this is not known to the normal user. So this extra step - user selects MIME type, which is internally somehow mapped to set of RDF formats is a bit confusing. 

As a result, instead of having 21 MIME types, we will have 9 RDF Formats values (as above). 

So changes which needs to be done: 
1) the line https://github.com/UnifiedViews/Plugins/pull/354/files#diff-e941664f7964363b337b6e89024a847cR199 can be changed to "return RDFFormat.XXX", where XXX is the value from the combo box, without any translation from MIME type to RDFFormat. 
2) the combo box should allow the value listed above in this comment.

**Comment by mbl-eea-sk** [comment]
Notices have been incorporated, retest please.

**Comment by tomas-knap** [comment]
Ad https://github.com/UnifiedViews/Plugins/pull/354#issuecomment-144663941, when the target RDF format is wrong it ends with still rather confusing error message probably due to wrong spacing of "Failed to convert file" and "eu.unifiedviews.dpu.exception": 

screen shot 2015-10-05 at 11 43 08

Please improve a bit the readability. 

Ad https://github.com/UnifiedViews/Plugins/pull/354#issuecomment-144669497,  thanks! 

Ad https://github.com/UnifiedViews/Plugins/pull/354#issuecomment-144663484, the desired solution is that "Type of Graph" is first config option on its own row. Then "Commit size" follows etc. So every config option is on its own row, "Type of Graph" is on the first row. Sorry if not being 100% precise about what should be done.

**Comment by tomas-knap** [comment]
Please update also SK version of About.md (documentation of RDF format is missing) based on EN version. Also take into account I changed the desc of the "Target RDF format" to "RDF format of input files", please reflect this also in resources_sk.properties.

Then can be merged.

**Pull request merged**